src/fragments/lib/geo/js/maps.mdx

```diff
@@ -75,7 +75,7 @@ body,
 To display markers on a map, use the `drawPoints` function. `drawPoints` expects:
 
 - `sourceName` - specifies the layer on which the markers are rendered on. You can edit existing markers by passing the same `sourceName`
-- coordinate data - the coordinate data of the markers to be displayed
+- coordinate data - (latitude, longitude) the coordinate data of the markers to be displayed
```

Comment:
Just want to reverse these because that's the way the type is set in the code.

```suggestion
- coordinate data - (longitude, latitude) the coordinate data of the markers to be displayed
```